### PR TITLE
Update stuff

### PR DESCRIPTION
**Pull request by jcmfernandes**
This gem seems abandoned.

Bring in support for `grape-entity >= 0.5.0` that is already in the project's repository `master` branch, and take care of some stuff that ruby and rails deprecated.

prime @jturkel 
cc @salsify/nazare-wave

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 rvm:
```
Comment (jcmfernandes) [inline .travis.yml:1]: I may want to remove this and add a circle config.

**Comment by jturkel** [comment]
I agree that the state of the upstream grape-entity-matchers is worrying since they haven't released an updated version of the gem in 3.5 years but they did merge our PR for `grape-entity >= 0.5.0` support in 2019 which is a positive sign. Ideally we could get off our fork rather than add more to it. Can you open a PR against the upstream repo with the deprecation fixes and request a new release of the gem?

Also for historical context, we typically let `master` track upstream `master`  for forked repos and create a `salsify-master` branch as necessary to accumulate changes. This approach makes it easier to keep track of what we've changed with the goal of getting off forks.

**Comment by jcmfernandes** [comment]
Oh, I didn't notice that Tim is an old colleague of ours 😅 thanks for the tip on the branches, I wasn't aware. I'm going to try to get this merged upstream.

**Comment by jcmfernandes** [comment]
@jturkel I already opened the PR upstream, but in the meanwhile, shall I try to get these changes merged here?

**Comment by jturkel** [comment]
I'm good getting these changes merged into `salsify-master` while we wait for the upstream changes 👍. Can we split this into two PRs: one that merges `master` into `salsify-master` (I've already updated `master` to track the latest version of `upstream/master`) and one that makes the actual changes to support Ruby 2.7? That will make it easier to understand the version history and review the appropriate changes.

**Comment by jcmfernandes** [comment]
Waiting for #6 to be merged.

BTW, it's forthcoming rails 6.1 that will break. As of today, without these changes, we only get deprecation warnings.

**Comment by jcmfernandes** [comment]
@jturkel your move 😉 .

**Review by jturkel — APPROVED** [review]

**Pull request merged**